saleor/product/models.py

```diff
@@ -98,6 +98,7 @@ def __repr__(self) -> str:
 
 class ProductType(ModelWithMetadata):
     name = models.CharField(max_length=128)
+    slug = models.SlugField(max_length=50, unique=True)
```

Comment:
`slug` fields should have max_length the same as fields they can be generated from.

Reply:
@maarcingebala  We should have at least a few more allowed characters in the slug field, for ower unique slug generator.

Reply:
I think it should be 4 or 5 more characters. One for `-` sign and 3 or 4 for digits.